README-v0.5.0.md

```diff
@@ -0,0 +1,713 @@
+(read, write, append, etc) on resources. WAC has several key features:
+
+1. The resources are identified by URLs, and can refer to any web documents or
+  resources.
+2. It is *declarative* -- access control policies live in regular web documents,
+  which can be exported/backed easily, using the same mechanism as you would
+  for backing up the rest of your data.
+3. Users and groups are also identified by URLs (specifically, by
+  [WebIDs](https://github.com/solid/solid-spec#identity))
+4. It is *cross-domain* -- all of its components, such as resources, agent
+  WebIDs, and even the documents containing the access control policies, can
+  potentially reside on separate domains. In other words, you can give access
+  to a resource on one site to users and groups hosted on another site.
+
+## Access Control List Resources
+
+In a system that uses Web Access Control, each web resource has a set of
+*Authorization* statements describing:
+
+1. Who has access to that resource (that is, who the authorized *agents* are)
+2. What types (or *modes*) of access they have
+
+These Authorizations are either explicitly set for an individual resource, or
+(more often) inherited from that resource's parent folder or container. In
+either case, the Authorization statements are placed into separate WAC
+documents called *Access Control List Resources* (or simply *ACLs*).
+
+### Containers and Inherited ACLs
+
+The WAC system assumes that web documents are placed in hierarchical containers
+or folders. For convenience, users do not have to specify permissions on each
+individual resource -- they can simply set default permissions on a container using a
+[`acl:default`](#default-inherited-authorizations) predicate and the exact container URI
+as the object, and have all
+of the resources in that container [inherit](#acl-inheritance-algorithm) those
+permissions.
+
+### Individual Resource ACLs
+
+For fine-grained control, users can specify a set of permissions for each
+individual resource (which overrides any permissions of its parent container).
+See the [Example WAC Document](#example-wac-document) section to get an idea
+for what that would look like.
+
+### ACL Resource Location Discovery
+
+Given a URL for an individual resource or container, a client can discover the
+location of its corresponding ACL by performing a `HEAD` (or a `GET`) request
+and parsing the `rel="acl"` link relation.
+
+Example request to discover the location of the ACL resource for a web document
+at `http://example.org/docs/file1` is given below:
+
+```http
+HEAD /docs/file1 HTTP/1.1
+Host: example.org
+
+HTTP/1.1 200 OK
+Link: <file1.acl>; rel="acl"
+```
+
+The request to discover the location of a container's ACL resource looks
+similar:
+
+```http
+HEAD /docs/ HTTP/1.1
+Host: example.org
+
+HTTP/1.1 200 OK
+Link: <.acl>; rel="acl"
+```
+
+Note that the `acl` link relation uses relative path URLs (the absolute URL of
+the ACL resource in the above example would be `/docs/.acl`).
```

Comment:
I think `/docs/.acl` would be called "a relative URL with an absolute path" and the absolute URL would be `http://example.org/docs/.acl`

Reply:
I'll stop reviewing this file now as you mentioned in https://gitter.im/solid/specification?at=60d1a4f96c992105fd926427